Makefile

```diff
@@ -0,0 +1,9 @@
+.PHONY: pytest lint
+
+pytest:	export PYTHONPATH=.
```

Comment:
lgtm

On Mon, Jul 11, 2016 at 6:42 PM, Alexandra L Johnson <
notifications@github.com> wrote:

> In Makefile
> <https://github.com/sigopt/sigopt-python/pull/61#discussion_r70366287>:
>
> > @@ -0,0 +1,9 @@
> > +.PHONY: pytest lint
> > +
> > +pytest:	export PYTHONPATH=.
>
> The tabs looks a lot uglier here than in sublime, is this a big deal?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/sigopt/sigopt-python/pull/61/files/6778002573d094ee4f303ccaad82e36e6584d809#r70366287>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAYmnIf6TZk6YKM4LvxLnPzNqTwWyLSJks5qUvEegaJpZM4JJ9Sy>
> .
>